docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md

```diff
@@ -253,6 +253,7 @@ The **RunConfiguration** element can include the following elements:
 |**TestAdaptersPaths**||One or more paths to the directory where the TestAdapters are located|
 |**MaxCpuCount**|1|This setting controls the degree of parallel test execution when running unit tests using available cores on the machine. The test execution engine starts as a distinct process on each available core, and gives each core a container with tests to run. A container can be an assembly, DLL, or relevant artifact. The test container is the scheduling unit. In each container, the tests are run according to the test framework. If there are many containers, then as processes finish executing the tests in a container, they're given the next available container.<br /><br />MaxCpuCount can be:<br /><br />n, where 1 <= n <= number of cores: up to n processes are launched<br /><br />n, where n = any other value: the number of processes launched can be up to the number of available cores. For instance, set n=0 to let the platform automatically decide the optimal number of processes to launch based on the environment.|
 |**TestSessionTimeout**||Allows users to terminate a test session when it exceeds a given timeout. Setting a timeout ensures that resources are well consumed and test sessions are constrained to a set time. The setting is available in **Visual Studio 2017 version 15.5** and later.|
+|**DotnetHostPath**||Specify a custom path to dotnet host that is used to run the testhost. This is useful when you are building your own dotnet, for example when building the dotnet/runtime repository. Specifying this option will skip looking for testhost.exe, and will always use the testhost.dll. 
```

Comment:
@nohwnd : I wonder if we want to document this. Would we need to support additional unplanned workflows with this? Would other teams apart from the dotnet team for instance need this...

Reply:
I don't know. Probably not. This is mainly aimed to the dotnet team because they are building their own dotnet. I already expressed my concerns about this being yet another config option in already complex thing. So I will happily keep this undocumented.